nextflow_schema.json

```
@@ -372,19 +372,7 @@
                     "type": "string",
                     "default": "all",
                     "description": "The NCBI taxonomic groups to download. Options include 'all', 'archaea', 'bacteria', 'fungi', 'invertebrate', 'metagenomes', 'plant', 'protozoa', 'vertebrate_mammalian', 'vertebrate_other', and 'viral'. A comma-separated list is also valid (e.g., 'bacteria,viral').",
-                    "enum": [
-                        "all",
-                        "archaea",
-                        "bacteria",
-                        "fungi",
-                        "invertebrate",
-                        "metagenomes",
-                        "plant",
-                        "protozoa",
-                        "vertebrate_mammalian",
-                        "vertebrate_other",
-                        "viral"
-                    ]
+                    "pattern": "^(?:[all|archaea|bacteria|fungi|invertebrate|metagenomes|plant|protozoa|vertebrate_mammalian|vertebrate_other|viral]+(?:,[all|archaea|bacteria|fungi|invertebrate|metagenomes|plant|protozoa|vertebrate_mammalian|vertebrate_other|viral]+)*)?$"
```

`[]` would match on a character level, so it matches also words made out of any them, e.g. human: https://regex101.com/r/5s7TUZ/1. Without them and the negative look-ahead, it should work as expected: https://regex101.com/r/9fLyRM/1
```suggestion
                    "pattern": "^((all|archaea|bacteria|fungi|invertebrate|metagenomes|plant|protozoa|vertebrate_mammalian|vertebrate_other|viral)?,?)*(?<!,)$"
```